src/fastmcp/server/server.py

```diff
@@ -205,6 +205,7 @@ def __init__(
             | None
         ) = None,
         tags: set[str] | None = None,
+        serializer: Callable[[Any], str] | None = None,
```

Comment:
Minor nit, but could we call this `tool_serializer` because (at least at this time) I think it only applies to tools as resources/prompts have much more fixed representations. I could imagine a different resource_serializer being specified for e.g. converting resources bytes to some other format in the future (not rushing to implement that!)

I think the arg is great staying as `serializer` on the tool / tool manager classes because obviously already scoped to tools, so this is only a comment for the server kwarg itself